docs/resources/cfw_firewall.md

```diff
@@ -68,6 +68,30 @@ resource "huaweicloud_cfw_firewall" "test" {
 }
 ```
 
+### firewall with IPS switch and IPS protection mode
+
+```hcl
+resource "huaweicloud_cfw_firewall" "test" {
+  name = "test"
+
+  flavor {
+    version = "Professional"
+  }
+
+  tags = {
+    key = "value"
+    foo = "bar"
+  }
+
+  charging_mode = "prePaid"
+  period_unit   = "month"
+  period        = 1
+  auto_renew    = false
+  ips_switch_status = 1
+  ips_protection_mode  = 1

```

Comment:
```suggestion
    ips_switch_status = 1
  ips_protection_mode  = 1
  
  charging_mode = "prePaid"
  period_unit   = "month"
  period        = 1
  auto_renew    = false
```
align the equal signs, and fix other places

Reply:
fixed